File: selectionSort.js
let start = performance.now();

function selection_sort(arr) {
    let counter = 0;
    while (counter < arr.length) { 
        shortest = arr[counter]
        position = counter
        for (let i = counter; i < arr.length; i++) { 
            if (arr[i] < shortest) {
                position = i;
                shortest = arr[i];
            }
        }
        let temp = arr[counter]
        arr[counter] = arr[position];
        arr[position] = temp;
        counter++;
    }
    return arr;
}

let end = performance.now();

console.log(`${selection_sort([33, 12, 2, 32, 0, 33, 39, 34, 12, 26])}, ${end - start} seconds`)

File: selection_sort.py
import time

start = time.time()
def selection_sort(arr):
    count = 0
    while count < len(arr):
        shortest = arr[count]
        position = count
        for i in range(count, len(arr)):
            if arr[i] < shortest:
                shortest = arr[i]
                position = i
        arr[count], arr[position] = arr[position], arr[count]
        count += 1
    return arr

print(selection_sort([33, 12, 2, 32, 0, 33, 39, 34, 12, 26]))

end = time.time()

total_time = end - start

print(f"Time to calculate was: {str(total_time)}")